src/cunumeric/mapper.cc

```diff
@@ -161,6 +161,13 @@ std::vector<StoreMapping> CuNumericMapper::store_mappings(
       mappings.back().policy.exact = true;
       return std::move(mappings);
     }
+    case CUNUMERIC_SEARCHSORTED: {
+      std::vector<StoreMapping> mappings;
+      auto& inputs = task.inputs();
+      mappings.push_back(StoreMapping::default_mapping(inputs[0], options.front()));
```

Comment:
what does this do?

Reply:
The next line `exact=true` is supposed to ensure that we always have dense data to ensure
`assert(input.accessor.is_dense_arbitrary(rect_base));`